internal/dataplane/kong_client.go

```diff
@@ -395,17 +395,17 @@ func (c *KongClient) Update(ctx context.Context) error {
 		}
 	}
 
-	c.logger.V(util.DebugLevel).Info("parsing kubernetes objects into data-plane configuration")
+	c.logger.V(util.DebugLevel).Info("Parsing kubernetes objects into data-plane configuration")
```

Comment:
https://github.com/Kong/kubernetes-ingress-controller/blob/13da44018b05d518a01f49bc10320f1c4b710871/internal/dataplane/kong_client.go#L185 
Here should be modified.